src/main/java/net/openhft/chronicle/threads/EventGroup.java

```diff
@@ -178,16 +175,16 @@ public EventGroup(final boolean daemon,
                     .collect(Collectors.toSet());
             core = priorities.stream().anyMatch(VanillaEventLoop.ALLOWED_PRIORITIES::contains)
                     ? corePriorities.equals(EnumSet.of(HandlerPriority.MEDIUM))
-                    ? new MediumEventLoop(this, name + "core-event-loop", pauser, daemon, binding)
-                    : new VanillaEventLoop(this, name + "core-event-loop", pauser, 1, daemon, binding, priorities)
+                    ? new MediumEventLoop(this, name + "/core-event-loop", pauser, daemon, binding)
```

Comment:
in some of our doco and examples we name event loops as "xxxx/" i.e. with a trailing slash. This may seem minor, but if you are going to add `/` here then we should remove the trailing slash from `name`

Reply:
We should probably omit the `/` if name is blank also? blank name is the default in the EventGroupBuilder

Reply:
I have moved the "/" related changes to a new PR